dockerized

```diff
@@ -5,8 +5,11 @@ set -euxo pipefail
 docker pull ubuntu:20.04
 
 # CACHEBUST means this will apply the updates once a day
-docker build . -t distributions --build-arg CACHEBUST=`date --iso-8601=date` --build-arg USER_UID=$(id -u "$USER")
+docker build . -t distributions \
+    --build-arg CACHEBUST=`date --iso-8601=date` \
+    --build-arg USER_UID=$(id -u "$USER") \
+    --build-arg GO_IPFS_VER=${GO_IPFS_VER:-$(curl -s https://dist.ipfs.io/go-ipfs/versions | tail -n 1)} # match http api client version on CI
```

Comment:
:information_source: this controls the version of the CLI client _inside_ `./dockerized` build, which talks to go-ipfs running _outside_ on the host. Here, we ensure it uses the same version (with fallback to the latest one if `GO_IPFS_VER` is undefined).

Reply:
Should this comment live in the source file?

Reply:
I believe it is good to have it here in case someone else tried to modify those scripts in the future   – it explains why we install go-ipfs twice (daemon + client in docker)